articles/connections/nativesocial/facebook.md

```diff
@@ -102,9 +102,11 @@ and the response from Auth0:
 }
 ```
 
-::: note
-Auth0 needs the user profile as part of the Token Exchange payload. This is because the Facebook Session Access Token cannot be used to directly retrieve the profile, and the Facebook Access Token cannot be sent directly to the server, because of [Apple's AppStore Review Guidelines](https://developer.apple.com/app-store/review/guidelines) policies. Therefore, it must be retrieved in the client and sent to Auth0 in this fashion.
-:::
+## User Profile and Email Validation
+
+In the previous example you had to retrieve the User Profile from Facebook and include it in the call to `/oauth/token`. This is because the Facebook Session Access Token cannot be used to directly retrieve the profile, and the Facebook Access Token cannot be sent directly to the server, because of [Apple's AppStore Review Guidelines](https://developer.apple.com/app-store/review/guidelines) policies. Therefore, it must be retrieved in the client and sent to Auth0 in this fashion.
```

Comment:
```suggestion
In the previous example, you had to retrieve the User Profile from Facebook and include it in the call to `/oauth/token`. This is because the Facebook Session Access Token cannot be used to directly retrieve the profile, and the Facebook Access Token cannot be sent directly to the server, due to [Apple's AppStore Review Guidelines](https://developer.apple.com/app-store/review/guidelines). Therefore, it must be retrieved in the client and sent to Auth0 in this fashion.
```